.travis.yml

```diff
@@ -12,33 +11,34 @@ arch:
 services:
   - docker  
 
+# Save up the cargo build cache
+cache: cargo  
+
 before_install:
 - rustup component add rustfmt
 
 # cargo test
 rust:
   - stable
   - beta
-  - nightly
 jobs:
-  allow_failures:
-    - rust: nightly
   fast_finish: true
 
-# Cargo fmt
-script:
-- cargo build
-- cargo test
-- cargo fmt -- --check
+# Build Test Fmt
+script: ./scripts/build.sh
+
+# Running deb build scripts
+script: ./rustscan-debbuilder/run.sh
 
 # Add in the github release details here
-# deploy:
-#   provider: releases
-#   api_key: "GITHUB OAUTH TOKEN"
-#   file: "FILE TO UPLOAD"
-#   skip_cleanup: true
-#   on:
-#     tags: true
+deploy:
+  provider: releases
+  api_key: $GH_0AUTH_TOKEN
+  file: target/debug/rustscan
```

Comment:
If this uploads the binary file itself, that doesn't work as it uses external dependencies which aren't added to the binary :)